Lib/glyphsLib/builder/groups.py

```diff
@@ -87,13 +88,41 @@ def to_ufo_groups(self):
             source.font.groups[name] = glyphs[:]
 
 
+def _to_ufo_groups(glyph, groups, glyph_group, side, other_side):
+    if not glyph_group.endswith(".RTL"):
+        # Traditional group
+        group = f"public.kern{side}.{glyph_group}"
+        if glyph.name not in groups[group]:
+            groups[group].append(glyph.name)
+        # Additional RTL group
+        # This will add lots of unused groups which we'll prune later
+        # but is necessary to implement G3 RTL kerning. See kerning.py
+        group = f"public.kern{other_side}.{glyph_group}.RTL"
+        if glyph.name not in groups[group]:
+            groups[group].append(glyph.name)
+    else:
+        # Additional RTL group
+        # This will add lots of unused groups which we'll prune later
+        # but is necessary to implement G3 RTL kerning. See kerning.py
+        if not glyph_group.endswith(".RTL"):
```

Comment:
this if branch is never hit, because you have already ascertained that glyph_group.endswith(".RTL") within this `else:` block...

Reply:
Right, that was a leftover of a previous version of this section. I removed the condition